myhpi/tenca_django/templates/tenca_django/manage_list.html

```diff
@@ -0,0 +1,92 @@
+{% extends "base.html" %}
+{% load tenca_extras %}
+{% load i18n %}
+
+{% block title %}
+	{% trans "Manage List" %}
+{% endblock %}
+
+{% block breadcrumbs %}
+	<li><span class="fa fa-angle-right" aria-hidden="true"></span>&nbsp;<a href="{% url "tenca_django:tenca_dashboard" %}">{% trans "Mailing lists" %}</a></li>
+	<li><span class="fa fa-angle-right" aria-hidden="true"></span>&nbsp;{% trans "Manage list" %}</li>
+{% endblock %}
+
+{% block content %}
+	<p>{% blocktrans %}
+		This is the admin view for the owners of {{ listname }}
+	{% endblocktrans %}</p>
+
+	<h3>{% trans "Invite Link" %}</h3>
+	{% trans "Share this link to invite new members:" %}
+	<div class="card mb-2">
+		<div class="card-body">
+			<a href="{{ invite_link }}">{{ invite_link }}</a>
+		</div>
+	</div>
+
+	<h3>{% trans "List options" %}</h3>
+    <form method="post">
+        {% csrf_token %}
+        <div class="form-group">
+            <div class="form-check">
+                <input type="checkbox" name="notsubscribed_allowed_to_post" class="form-check-input" id="id_notsubscribed_allowed_to_post" checked="">
+                <label class="form-check-label" for="id_notsubscribed_allowed_to_post">
```

Comment:
I usually don't like duplicating the structure of the form into the HTML. in my other project, we used Crispy Forms as a replacement for the bootstrap4 stuff. What do you think about that? Maybe it's not worth including an additional dependency for such a small number of forms

Reply:
Could also be a separate issue